include/json/config.h

```diff
@@ -118,6 +120,12 @@
 #  define JSON_USE_INT64_DOUBLE_CONVERSION 1
 #endif
 
+// If non-zero, the library zeroes any memory that it has allocated before
+// it frees its
+#ifndef JSON_USE_SECURE_MEMORY
+#define JSON_USE_SECURE_MEMORY 1
```

Comment:
`0` by default -- at least until we measure the runtime/memory cost of using this allocator.